UnitystationLauncher/Models/ServerManager.cs

```diff
@@ -78,6 +84,25 @@ private async Task<IReadOnlyList<ServerViewModel>> GetServerList()
             return newServerList;
         }
 
+        private bool HasTrustedUrlSource(Server server)
```

Comment:
This would be nicer as a boolean property of `Server`

Reply:
done